barretenberg/cpp/pil/vm2/bytecode/bc_hashing.pil

```diff
@@ -15,15 +15,16 @@ include "../poseidon2_hash.pil";
 // Finally, we copy through a lookup/permutation based on the selector these field elements
 // to here (bc_hashing.pil) and then proceed to hashing.
 
-//   pc_index | bytecode_id | packed_field | incremental_hash |   output_hash      | latch |
-// -----------+-------------+--------------+------------------+--------------------+--------
-//      0     |      0      |     0xabc    |        len       | H(0xabc,len) = 0x2 |   0
-//     31     |      0      |     0x123    |        0x2       | H(0x123,0x2) = 0x3 |   0
-//     62     |      0      |     0x62d    |        0x3       | H(0x62d,0x3) = end |   1
-//     0      |      1      |     0x4ab    |        len       | H(0x4ab,len) = 0x5 |   0
-//     31     |      1      |     0x21f    |        0x5       | H(0x21f,0x5) = 0x6 |   0
-//     62     |      2      |     0x12a    |        0x6       | H(0x12a,0x6) = 0x9 |   0
-//     93     |      2      |     0x982    |        0x9       | H(0x982,0x9) = end |   1
+// bc_hash_0 = H([sep, 0x111, 0x222, 0x333, 0x444, 0x555, 0x666, 0x777])
+// bc_hash_1 = H([sep, 0xaaa, 0xbbb, 0xccc])
+// p_f = packed_fields:
+//   pc_index |  pc_index_1 |  pc_index_2 | bytecode_id |  p_f_0  |  p_f_1  |  p_f_2  |  output_hash  | latch | START | padding |
```

Comment:
I was surprised that `output_hash` was always the same. I was thinking: how are the different rows of the hash being connected? could they be reordered? etc.

Then when looking at the poseidon2_hash pil I realized that it is exactly the propagation of that hash to all rows (in that gadget) + "infeasible to crack" which makes it secure. Maybe it would be good to at least remind (closer to the beginning of the file) that this is a quirk or the poseidon2_hash gadget (/remind how it works).